gprofiler/profilers/__init__.py

```diff
@@ -1,8 +1,13 @@
 # NOTE: Make sure to import any new process profilers to load it
-from gprofiler.profilers.java import JavaProfiler
-from gprofiler.profilers.perf import SystemProfiler
-from gprofiler.profilers.php import PHPSpyProfiler
 from gprofiler.profilers.python import PythonProfiler
-from gprofiler.profilers.ruby import RbSpyProfiler
+from granulate_utils import is_windows
+if not is_windows():
+    from gprofiler.profilers.java import JavaProfiler
+    from gprofiler.profilers.perf import SystemProfiler
+    from gprofiler.profilers.php import PHPSpyProfiler
+    from gprofiler.profilers.ruby import RbSpyProfiler
+from gprofiler.profilers.python_ebpf import PythonEbpfProfiler
```

Comment:
This one needs to be under `if is_windows()` as well, no?

Reply:
No, because PythonProfiler(ProfilerInterface) uses it and is required for both Windows and Linux

Reply:
Was split out, resolving